docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/MapKeyToVectorMultiColumn.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace Samples.Dynamic
+{
+    class MapKeyToVectorMultiColumn
+    {
+        /// This example demonstrates the use of MapKeyToVector by mapping keys to floats[] for multiple columns at once. 
+        /// Because the ML.NET KeyType maps the missing value to zero, counting of uints starts at 1, so the values
+        /// converted to KeyTypes will appear skewed by one. See  https://github.com/dotnet/machinelearning/issues/3072
+        public static void Example()
```

Comment:
i have tried to add explanations, because of issue 3072. 
I think we should change the mapping of the missing value to 0...